.travis.yml

```diff
@@ -31,8 +31,8 @@ branches:
 
 env:
   - 
-  - BACKEND="OPENMP"
-#  - BACKEND="PTHREAD"
+#  - BACKEND="OPENMP"
+  - BACKEND="PTHREAD"
```

Comment:
???
Did you mean to disable all the OpenMP builds?

Reply:
No, we still have some of them testing `OpenMP` (the ones that also do the coverage).

Reply:
No AppleClang OpenMP for instance.  I guess my question is did you guys carefully look at what this change means

Reply:
But `AppleClang` with `OpenMP` is disabled anyway: https://github.com/kokkos/kokkos/blob/b5b7660af56d883ffc8892fa8a253ee257f38e26/.travis.yml#L43-L57

For me, this pull request still is an improvement.